rise-data.html

```diff
@@ -51,6 +51,14 @@
       }
     }
 
+    function supportsSessionStorage() {
```

Comment:
@rodrigopavezi I'm thinking we can remove the "supports" methods for local and session storage as browser compatibility is across the board supported including Android ^2.1. Thoughts?